website/addons/owncloud/utils.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from website.oauth.models import ExternalAccount
+from website.security import encrypt, decrypt
+
+
+class ExternalAccountConverter:
```

Comment:
@mfraezz and I discussed this.  Please make it part of this PR, otherwise we won't have any real-world use cases to test it with.